packages/front-end/components/Radix/Badge.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Badge as RadixBadge } from "@radix-ui/themes";
+import { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+import { RadixColor } from "@/components/Radix/HelperText";
+
+type Props = {
+  content: string;
+  color?: RadixColor;
+  variant?: "solid" | "soft" | "surface" | "outline";
+  highContrast?: boolean;
+  radius?: "none" | "small" | "medium" | "large" | "full";
+} & MarginProps;
+
+export default function Badge({
+  content,
+  color,
+  variant,
+  highContrast,
+  radius,
+  ...containerProps
+}: Props) {
+  return (
+    <Flex {...containerProps}>
+      <RadixBadge
+        color={color}
+        variant={variant}
+        highContrast={highContrast}
+        radius={radius}
+      >
+        {content}
```

Comment:
Is there a reason we need to wrap this in `<Flex>`?  Also, I think I prefer using `children` instead of a custom `content` prop here.

Reply:
Children is cool. For flex, I followed the examples here: https://www.radix-ui.com/themes/docs/components/badge

Reply:
Those examples are confusing, but I think they are just using Flex since they are showing multiple example badges next to each other and want to add spacing between them.  A single badge on its own doesn't need to be wrapped in Flex